src/helpers/app-store.tsx

```diff
@@ -3,15 +3,18 @@ import React, { createContext, useContext, useState, ReactNode } from 'react';
 interface AppContextType {
   username: string;
   setUsername: (name: string) => void;
+  role: string;
```

Comment:
you can define values as a type in TS, so the user should enter only these types
`role: 'admin' | 'user';`